code/game/objects/effects/effects.dm

```diff
@@ -63,3 +63,57 @@
 
 /obj/effect/dummy/singularity_act()
 	return
+
+/// Proc used for 96x96 effects such as runes(Nar-sie, Apocalypse, Heretic and Wizard ones). So they don't overlay above the items.
+/obj/effect/proc/overlay_for_96x96_effects()
+	var/mutable_appearance/bottom = mutable_appearance(icon, icon_state)
+	bottom.pixel_x = pixel_x
+	bottom.pixel_w = -(pixel_x)
+	bottom.add_filter("mask", 1, alpha_mask_filter(icon = icon(icon, "row_mask_bottom")))
+	add_overlay(bottom)
```

Comment:
it would be possible to do this effect without needing to copy over the icon and state at the moment of this proc being called.
We could use render_source/render_target to directly copy the appearance of the base rune, and relay it how we'd like.

This would avoid issues with the icon changing, or going through an animation, and breaking the effect. It is slightly more client heavy, but this is already not great for that sort of thing.